python-converter/binary-converter.py

```diff
@@ -1,11 +1,191 @@
-def main():
-    response = ''
+"""
+    # BINARY CONVERTER
+    * AUTHOR: afmireski (André Felipe Mireski)
+"""
+
+def get_bits_len(number: int) -> int:
```

Comment:
Lint problem, 2 spaces required
```suggestion
"""


def get_bits_len(number: int) -> int:
```